### PR TITLE
Subpage titles

### PR DESCRIPTION
**Pull request by dasyad00**
I'll do the titles for board subpage tonight

### CODE REVIEW

File: cosmos_cms/migrations/0004_committee_subpage_title.py [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+import django.db.models.deletion
+
+
+class Migration(migrations.Migration):
+
+    replaces = [
+        ("cosmos_cms", "0004_committeesubpagetitlepluginmodel"),
+        ("cosmos_cms", "0005_auto_20200826_1138"),
+        ("cosmos_cms", "0006_committeesubpagetitlepluginmodel_image"),
+    ]
```
Comment (bcvandendool) [inline cosmos_cms/migrations/0004_committee_subpage_title.py:13]: Remove these after squashing

**Review by bcvandendool — CHANGES_REQUESTED** [review]
Please use django-pipeline as can be seen in development

**Comment by maximekabel** [comment]
The plugin asks to choose a committee but then it doesn't actually do anything with that model. It would be better if you have the plugin pull the name and the description from the committee model. Also right now you can upload any picture which means that it can come out too tall. I would recommend you force a certain aspect ratio when uploading.

**Review by bcvandendool — CHANGES_REQUESTED** [review]
Please also add the images of the board/committee. For example the image to the left, and to the right the name of the board/committee with the description underneath.

**Comment by dasyad00** [comment]
> Please also add the images of the board/committee. For example the image to the left, and to the right the name of the board/committee with the description underneath.

can you elaborate what you mean by this? I can't seem to find an example on the website https://punintendedinsta.wixsite.com/uwuwhatsthis/committee-subpage-template-v1

File: cosmos/templates/base.html [hunk 2]
```diff
@@ -13,6 +13,7 @@
           integrity="sha384-JcKb8q3iqJ61gNV9KGb8thSsNjpSL0n8PARn9HuZOnIxN0hoP+VmmDGMN5t9UJ0Z" crossorigin="anonymous">
     {# Custom CSS #}
     {% stylesheet "cosmos" %}
+    {% stylesheet "comm_title" %}
```
Comment (bcvandendool) [inline cosmos/templates/base.html:16]: I dont think this is needed, is it?

**Review by bcvandendool — CHANGES_REQUESTED** [review]
Spacing is fucked, its wider than the screen rn, and with container md it is also wider than the rest, something needs to be set different with the padding and margin. Furthermore can you try a 1:2 ratio of image:text, and see how that looks? i think that might look a bit better, not sure though.

**Pull request merged**